en/mozillavpn.xliff

```diff
@@ -0,0 +1,556 @@
+    </trans-unit>
+    <trans-unit id="vpn.postAuthentication.continue">
+      <source xml:space="preserve">Continue</source>
+      <context-group purpose="location"><context context-type="linenumber">93</context></context-group>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/ui/states/StateSubscriptionNeeded.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="vpn.iap.subscriptionNeeded">
+      <source xml:space="preserve">Subscription needed</source>
+      <context-group purpose="location"><context context-type="linenumber">19</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.iap.subscribeNow">
+      <source xml:space="preserve">Subscribe now</source>
+      <context-group purpose="location"><context context-type="linenumber">25</context></context-group>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/ui/views/ViewDevices.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="vpn.devices.deviceAccessibleName">
+      <source xml:space="preserve">%1 %2</source>
+      <context-group purpose="location"><context context-type="linenumber">71</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.devices.currentDevice">
+      <source xml:space="preserve">Current Device</source>
+      <context-group purpose="location"><context context-type="linenumber">129</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.devices.addedltHour">
+      <source xml:space="preserve">Added less than an hour ago</source>
+      <context-group purpose="location"><context context-type="linenumber">134</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.devices.addedXhoursAgo">
+      <source xml:space="preserve">Added a few hours ago (%1)</source>
+      <context-group purpose="location"><context context-type="linenumber">138</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.devices.addedXdaysAgo">
+      <source xml:space="preserve">Added %1 days ago</source>
+      <context-group purpose="location"><context context-type="linenumber">141</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.devices.removeA11Y">
+      <source xml:space="preserve">Remove %1</source>
+      <context-group purpose="location"><context context-type="linenumber">172</context></context-group>
+      <note annotates="source" from="developer">"This is a label to make the icon button accessible</note>
+    </trans-unit>
+    <trans-unit id="vpn.devices.removeDeviceQuestion">
+      <source xml:space="preserve">Remove device?</source>
+      <context-group purpose="location"><context context-type="linenumber">389</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.devices.deviceRemovalConfirm">
+      <source xml:space="preserve">Please confirm you would like to remove\n%1.</source>
```

Comment:
Please add a comment explaining what %1 is (I assume the device's name), and why there's a "\n" in there, e.g.

> %1 is the name of the device being removed. "\n" is used to display it on a new line, please keep it in the translation.